apps/asap-server/test/fixtures/teams.fixtures.ts

```diff
@@ -251,7 +256,13 @@ export const getGraphqlTeam = (
     ],
     tools,
   },
-  referencingUsersContents: [getGraphQLUser(undefined, id)],
+  referencingUsersContents: [
+    getGraphQLUser({
+      flatData: {
+        teams: [getGraphQLUserTeam({ id: [getGraphQLUserTeamId({ id })] })],
+      } as GraphQLUserTeamFlatData,
```

Comment:
you shouldnt need the cast?

Reply:
Yes. getGraphQLUser takes a partial NonNullable<FetchUserQuery['findUsersContent']> - from my understand partial only works on a shallow level. Here we want to pass in a Partial flatData.